### PR TITLE
Filter - Display "Empty" Result

### PR DESCRIPTION
**Pull request by mantis**

### CODE REVIEW

File: src/mui/list/List.js [hunk 1]
```diff
@@ -209,21 +213,27 @@ export class List extends Component {
...
+                            })}
+                            { pagination && React.cloneElement(pagination, {
+                                total,
+                                page: parseInt(query.page, 10),
+                                perPage: parseInt(query.perPage, 10),
+                                setPage: this.setPage,
+                            }) }
+                        </div>
+                        :
+                        <cardText style={styles.noResults}>{translate('aor.list.no_results')}</cardText>
```
Comment (djhi) [inline src/mui/list/List.js]: `cardText` should be written `CardText` and the import is missing

**Comment by fzaninotto** [comment]
Can you PR against the `next` release? `master` is only for bugfixes.

File: src/i18n/messages.js [hunk 2]
```diff
@@ -61,5 +61,8 @@ export default {
         validation: {
             required: 'Required',
         },
+		list: {
```
Comment (fzaninotto) [inline src/i18n/messages.js]: Maybe it's GitHub, or maybe there are mixed tabs and spaces here

File: src/mui/list/List.js [hunk 3]
```diff
@@ -17,6 +17,10 @@ import translate from '../../i18n/translate';
 
 const filterFormName = 'filterForm';
 
+const styles = {
+    noResults: { padding: 20, display: 'flex', alignItems: 'flex-end' },
```
Comment (fzaninotto) [inline src/mui/list/List.js]: can we avoid `flex` on this? Normally, all flex styles should be prefixed, and it's cumbersome.

File: src/mui/list/List.spec.js [hunk 4]
```diff
@@ -21,6 +21,38 @@ describe('<List />', () => {
...
+                {...defaultProps}
+                translate={translateSpy}
+                children={[]}
+                total={0}
+            >
+                <div />
+            </List>
+        );
+
+        assert.equal(translateSpy.calledWith('aor.list.no_results'), true);
```
Comment (fzaninotto) [inline src/mui/list/List.spec.js]: I'd prefer that you use the `ShallowWrapper` and `findElement` to test the result. Your current test is testing the implementation.

File: src/i18n/messages.js (hunk 2 above)
Comment (fzaninotto) [inline src/i18n/messages.js]: I'd put that under the `navigation` key

File: src/mui/list/List.js (hunk 1 above)
Comment (mantis) [inline src/mui/list/List.js]: heh - fixed

File: src/i18n/messages.js (hunk 2 above)
Comment (mantis) [inline src/i18n/messages.js]: fixed and fixed

File: src/mui/list/List.js (hunk 3 above)
Comment (mantis) [inline src/mui/list/List.js]: I dont think that display: 'flex', alignItems: 'flex-end' is needed - the padding 20 I thought kept it roughly inline with the table header it was replacing

**Comment by mantis** [comment]
*sigh*, that didn't quite go to plan... guess this PR just moved to #487

**Pull request closed without merging**